lib/analysis/channel.py

```diff
@@ -7,389 +7,385 @@
 
 def conv_len_conv_refr_time(log_dict, nicks, nick_same_list, rt_cutoff_time, cutoff_percentile):
```

Comment:
@bhuvangupta008 Should these changes be made in this PR?

Reply:
the changes in the channel module are:

- conversion of tab indentations to 4 spaces
- minor change in the line breaks of different code lines based on the Pep8 recommendations
- **No change to logic/lines of code**

Reply:
Is it not unrelated to this PR? Maybe they should be in a new commit.